src/interface.c

```diff
@@ -28602,6 +28605,22 @@ int hashconfig_init (hashcat_ctx_t *hashcat_ctx)
                  hashconfig->dgst_pos2      = 2;
                  hashconfig->dgst_pos3      = 3;
                  hashconfig->st_hash        = ST_HASH_18400;
+                 hashconfig->st_pass        = ST_HASH_18500;
```

Comment:
@s3inlc, ouch! :laughing:
GH-1821

Reply:
Well, the tests for 18500 run just fine. It was the 18400 case branch that took the hit :sweat_smile: